### PR TITLE
chore(utilities): add breakpoint options to docs

### LINKED ISSUE #3702: Review utility documentation
Reported by @srambach, the breakpoint suffixes are missing from the text/font-sizes utility class documentation. Let's review all of the utility docs and make sure everything is up to date.

### PR DESCRIPTION
**Pull request by srambach**
Fixes #3702 

Adds documentation about breakpoints to the text and background color utilities and consolidates some repeated options.

### CODE REVIEW

File: src/patternfly/utilities/Text/examples/Text.md [hunk 1]
```diff
@@ -207,56 +208,33 @@ These text utilities can be used to modify text within the PatternFly framework.
...
-| `.pf-u-icon-color-light`    | `*`        | Sets font-color to light icon color    |
-| `.pf-u-icon-color-dark`     | `*`        | Sets font-color to dark icon color     |
-| `.pf-u-text-break-word`     | `*`        | Sets word-break to break-word          |
-| `.pf-u-text-nowrap`         | `*`        | Sets white-space to nowrap             |
-| `.pf-u-text-wrap`           | `*`        | Sets white-space to normal             |
-| `.pf-u-text-truncate`       | `*`        | Truncates text field                   |
+| Class                                                                  | Applied to | Outcome                                             |
+| ---------------------------------------------------------------------- | ---------- | --------------------------------------------------- |
+| `.pf-u-font-size-{xs, sm, md, lg, xl, 2xl, 3xl, 4xl}{-on-[breakpoint]}`| `*`        | Sets font-size to xs, sm, md, lg, xl, 2xl, 3xl, 4xl |
+| `.pf-u-font-weight-{light, normal, bold}{-on-[breakpoint]}`            | `*`        | Sets font-size to light, normal, bold               |
```
Comment (mcoker) [inline src/patternfly/utilities/Text/examples/Text.md:218]: nit - missing a conjunction
```suggestion
| `.pf-u-font-weight-{light, normal, bold}{-on-[breakpoint]}`            | `*`        | Sets font-size to light, normal, or bold               |
```

File: src/patternfly/utilities/Text/examples/Text.md [hunk 2]
```diff
@@ -207,56 +208,33 @@ These text utilities can be used to modify text within the PatternFly framework.
...
-| `.pf-u-disabled-color-200`  | `*`        | Sets font-color to disabled color 200  |
-| `.pf-u-icon-color-light`    | `*`        | Sets font-color to light icon color    |
-| `.pf-u-icon-color-dark`     | `*`        | Sets font-color to dark icon color     |
-| `.pf-u-text-break-word`     | `*`        | Sets word-break to break-word          |
-| `.pf-u-text-nowrap`         | `*`        | Sets white-space to nowrap             |
-| `.pf-u-text-wrap`           | `*`        | Sets white-space to normal             |
-| `.pf-u-text-truncate`       | `*`        | Truncates text field                   |
+| Class                                                                  | Applied to | Outcome                                             |
+| ---------------------------------------------------------------------- | ---------- | --------------------------------------------------- |
+| `.pf-u-font-size-{xs, sm, md, lg, xl, 2xl, 3xl, 4xl}{-on-[breakpoint]}`| `*`        | Sets font-size to xs, sm, md, lg, xl, 2xl, 3xl, 4xl |
```
Comment (mcoker) [inline src/patternfly/utilities/Text/examples/Text.md:217]: nit - missing a conjunction
```suggestion
| `.pf-u-font-size-{xs, sm, md, lg, xl, 2xl, 3xl, 4xl}{-on-[breakpoint]}`| `*`        | Sets font-size to xs, sm, md, lg, xl, 2xl, 3xl, or 4xl |
```

**Review by mcoker — CHANGES_REQUESTED** [review]

**Review by mattnolting — APPROVED** [review]
LPTM🚀

**Review by mcoker — APPROVED** [review]
🤩

**Pull request merged**